editor/plugins/tile_set_editor_plugin.cpp

```diff
@@ -2417,7 +2417,9 @@ void TileSetEditor::draw_polygon_shapes() {
 							colors.push_back(c_bg);
 						}
 					}
-					if (polygon.size() > 2) {
+					if (polygon.size() == 0)
+						continue;
+					else if (polygon.size() > 2) {
```

Comment:
This `else` is not necessary, as the `continue` just skips to the next iteration.

Reply:
Yes, thanks for pointing that out!